data/remnant/remnant missions.txt

```diff
@@ -2245,6 +2245,417 @@ mission "Remnant: Face to Maw 2B"
+			label work
+			`	"Our researchers have decided that we need to test how a ship performs with one of these keystones compared to a regular key stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the quantum keystone built with Hai crystals for a quantum key stone made from our crystals. Then we will go back to Peragenor and perform another scan. By using the same ships for both tests, we should be able to get the best data."`
+			choice
+				`	"Wait, you are a pilot?"`
+				`	"Okay."`
+					accept
+			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of a general education. It is especially worthwhile for those of us who work directly with starships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures suggest both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`
+				accept
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling (Keystone)" "Winds of Light"
+		dialog
+			`The scanner chimes softly, informing you that it has completed scanning the Winds of Light and has successfully archived the data.`
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the key stone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling" "Winds of Light"
+		dialog
+			`The scanner chimes softly once again, informing you that it has completed scanning the Winds of Light and has successfully archived the data from this second scanning.`
+	to offer
+		has "Remnant: Crystal Research 5: done"
+	on offer
+		conversation
+			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a key stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the keystone to a lift and disconnects it from the ship. As the keystone is hauled up to the roof another lift lowers the replacement key stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
```

Comment:
```suggestion
			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a keystone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the keystone to a lift and disconnects it from the ship; another lift soon lowers the replacement keystone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
```

Reply:
In this scene there is actually both a keystone and a key stone present. The keystone is being removed and lifted off, while the replacement key stone is being lowered into place.

Reply:
Also, in terms of replacing the "`As the keystone is hauled up to the roof another lift lowers the replacement`" with "`another lift soon lowers the replacement`", the idea was to subtly convey the idea that the Remnant have their shipyards set up for fast turn around. The Remnant don't have much "defense in depth" to give them time to repair or refit ships. They need to be able to scramble and get ships fixed up and shipped out as fast as they possibly can; and part of that means that their rigging is setup so that replacement parts are already arriving while the old parts are still in the process of being removed. In other words, simultaneous operations.

I didn't want to *force* player realization of this fact, but I do want the details to be present so that they *can* notice it.

Reply:
I think the distinction between a "keystone" and a "keystone" is likely to be lost on most players and considered a typo--as I did. :)

OK on the other point.